user-guide/monitoring/vm_monitoring.rst

```diff
@@ -32,7 +32,7 @@ Dynamic Assignment of Monitoring Templates and Host Groups
 
 * **MON_ZABBIX_HOSTGROUPS_VM** - List of other existing Zabbix host groups, which will be used for all monitored servers in a virtual data center. Available placeholders are:
 
-    * ``{ostype}`` - ID type of the operating system (1 - *Linux*, 2 - *SunOS*, 3 - *BSD*, 4 - *Windows*, 5 - *SunOS Zone*).
+    * ``{ostype}`` - ID type of the operating system (1 - *Linux VM*, 2 - *SunOS VM*, 3 - *BSD VM*, 4 - *Windows VM*, 5 - *SunOS Zone*).
     * ``{ostype_text}`` - The type of the operating system with lowercase letters and spaces replaced with underscores (*linux*, *sunos*, *bsd*, *windows*, *sunos_zone*).
```

Comment:
The change in erigones/esdc-ce#1 also affects `vm.ostype_text()`, e.g. `bsd` will be `bsd_vm`